portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,36 +44,51 @@
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
```

Comment:
nit: for consistency, code blocks such as functions should not start with a blank line.